test/e2e/play_kube_test.go

```diff
@@ -2527,4 +2527,61 @@ invalid kube kind
 		Expect(inspect).Should(Exit(0))
 		Expect(inspect.OutputToString()).To(ContainSubstring(`map[]`))
 	})
+
+	It("podman play kube teardown", func() {
```

Comment:
Can we get a few more tests - teardown on a YAML that was never played, teardown on a YAML that was already stopped?